components/director/internal/domain/application/fixtures_test.go

```diff
@@ -14,7 +14,8 @@ import (
 )
 
 var (
-	testURL = "https://foo.bar"
+	testURL  = "https://foo.bar"
+	intSysID = "int-sys"
```

Comment:
maybe it is worth to put something in more uuid-like format here to indicate that this is the uuid.